lib/Alchemy/Phrasea/Controller/Prod/EditController.php

```diff
@@ -353,16 +353,14 @@ public function applyAction(Request $request) {
      */
     public function applyJSAction(Request $request): JsonResponse
     {
-        // todo : by worker
-
-        // for now call record_adapter. no check, no acl, ...
         /** @var stdClass $arg */
         $arg = json_decode($request->getContent());
+        $databoxId = reset($arg->records)->sbas_id;
```

Comment:
~~no. even if we "know" that sbas_id is always the same, this may change later.
since "records" is part of body, we should pass null as last arg.~~
correction :
It seems that the existing worker code relies heavily on a unique databox...
So we must change the json to receive a unique sbid, and an array of rids.

Reply:
ok , I've change that